go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/creack/pty v1.1.21
-	github.com/digitalocean/godo v1.131.0
+	github.com/digitalocean/godo v1.131.1-0.20241127211050-a97e39731918
```

Comment:
:wave: Could we use [v1.131.0](https://github.com/digitalocean/godo/releases/tag/v1.131.0) as required changes already should be there :pray: 

> https://github.com/digitalocean/godo/pull/759 - @imaskm - Add reserved ipv6 changes as Beta

Reply:
Nice :raised_hands:  Thanks for explanation :+1: